core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -906,6 +906,12 @@
     sa.config-updater.api.ci.config:
       bw_item: build_farm
       attachment: sa.config-updater.api.ci.config
+    build02_github_client_secret:
```

Comment:
Can you move it up so it comes after the `build01` one?

Reply:
I can.
I actually move it down here from the upper place.
I will soon add the kube files for SAs and thought all build02 items should be grouped together. :)

Reply:
Updated.